.github/workflows/pr_tests.yml

```diff
@@ -130,12 +130,41 @@ jobs:
       - run: 'bundle exec rake spec'
         continue-on-error: ${{matrix.puppet.experimental}}
 
-#  dump_contexts:
-#    name: 'Examine Context contents'
-#    runs-on: ubuntu-latest
-#    steps:
-#      - name: Dump contexts
-#        env:
-#          GITHUB_CONTEXT: ${{ toJson(github) }}
-#        run: echo "$GITHUB_CONTEXT"
-#
+  acceptance:
+    runs-on:
+      - ubuntu-latest
+    strategy:
+      matrix:
+        node:
+          - docker_alma8
+          - docker_alma9
+          - docker_centos9
+          - docker_oel8
+          - docker_oel9
+          - docker_rhel8
+          - docker_rhel9
+          - docker_rocky8
+          - docker_rocky9
+      fail-fast: false
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v2
+      - name: setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+      - name: install bundler
+        run: |
+          gem install bundler
+          bundle update
+      - name: install podman
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y podman
```

Comment:
Please test if this is actually necessary.  I think it might be included already on the GHA runners.